test/UID2.Client.Test/EncryptionTestsV3.cs

```diff
@@ -26,13 +26,15 @@ public void SmokeTest()
             var client = new UID2Client("ep", "ak", CLIENT_SECRET, IdentityScope.UID2);
             var refreshResult = client.RefreshJson(KeySetToJson(MASTER_KEY, SITE_KEY));
             Assert.True(refreshResult.Success);
+            
             string advertisingToken = UID2TokenGenerator.GenerateUid2TokenV3(EXAMPLE_UID, MASTER_KEY, SITE_ID, SITE_KEY, UID2TokenGenerator.DefaultParams);
             var res = client.Decrypt(advertisingToken, NOW);
             Assert.True(res.Success);
             Assert.Equal(EXAMPLE_UID, res.Uid);
+            Assert.Equal(IdentityType.Email, res.IdentityType);
```

Comment:
can you add a new test using a mobile-generated raw UID2 for testing IdentityType.Mobile ?